spec/requests/diary_entries_request_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "DiaryEntries", type: :request do
```

Comment:
I would remove this spec and cover the functionality in `type: :feature` one or multiple.

Reply:
Ok, can take that approach